README.rst

```diff
@@ -0,0 +1,50 @@
+Google Auth Python Library (Experimental)
+=========================================
+
+.. image:: https://travis-ci.org/GoogleCloudPlatform/google-auth-library-python.png?branch=master
+        :alt: Build status on Travis
+        :target: https://travis-ci.org/GoogleCloudPlatform/google-auth-library-python
+
+.. image:: https://readthedocs.org/projects/google-auth/badge/?version=latest
+        :alt: Documentation Status
+        :target: https://google-auth.readthedocs.io/en/latest/
+
+This library simplifies using Google's various authentication mechanisms to
+access Google APIs.
+
+Installing
+----------
+
+You can install directly from `GitHub <https://github.com/GoogleCloudPlatform/google-auth-library-python>`_::
+
+    $ pip install git+https://github.com/GoogleCloudPlatform/google-auth-library-python.git
+
+This package is not currently available from PyPI because it is experimental.
+
+Documentation
+-------------
+
+Google Auth Python Library has usage and reference documentation at `google-auth.readthedocs.io <https://google-auth.readthedocs.io>`_.
+
+Maintainers
+-----------
+
+- `@jonparrott <https://github.com/jonparrott>`_ (Jon Wayne Parrott)
+- `@dhermes <https://github.com/dhermes>`_ (Danny Hermes)
+- `@bjwatson <https://github.com/bjwatson>`_ (Brian Watson)
```

Comment:
That's fair, but I've noticed that people find projects more approachable if they're not faceless. I'll revisit this before finalizing all of the documentation.